content/04.baselines.md

```diff
@@ -15,3 +15,5 @@ It can be tempting to train baseline models with these conventional methods usin
 Hu and Greene [@doi:10.1101/385534] discuss a "Continental Breakfast Included" effect by which unequal hyperparameter tuning for different learning algorithms skews the evaluation of these methods, especially when the performance of an algorithm varies substantially with modest changes to its hyperparameters.
 Those wishing to compare different learning algorithms should tune the settings of both traditional and DL-based methods to optimize performance before making claims about relative performance differences.
 The performance comparison among DL models and many other ML approaches is informative only when the models are similarly well-tuned.
+
+To sum this tip up, create and fully tune several traditional models such as logistic regressors or random forests before implementing a DL model.
```

Comment:
I think you meant "regression models" instead of "regressors" (in stats, regressors refers to what is called "inputs features" in ML)